lua/wikis/commons/Widget/Match/StreamsContainer.lua

```diff
@@ -0,0 +1,54 @@
+---
+-- @Liquipedia
+-- page=Module:Widget/Match/StreamsContainer
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local Array = Lua.import('Module:Array')
+local Class = Lua.import('Module:Class')
+
+local StreamLinks = Lua.import('Module:Links/Stream')
+
+local Widget = Lua.import('Module:Widget')
+local MatchStream = Lua.import('Module:Widget/Match/Stream')
+
+---@class MatchStreamsContainer: Widget
+---@operator call(table): MatchStreamsContainer
+local MatchStreamsContainer = Class.new(Widget)
+MatchStreamsContainer.defaultProps = {
+	callToActionLimit = 0,
+}
+
+---@return Widget?
+function MatchStreamsContainer:render()
+	local streams = self.props.streams
+	if not streams then
+		return nil
+	end
+
+	local processedStreams = Array.flatMap(StreamLinks.countdownPlatformNames, function(platform)
+		return Array.map(streams[platform] or {}, function(stream)
+			return {platform = platform, stream = stream}
+		end)
+	end)
+
+	local numberOfStreams = #processedStreams
+	if numberOfStreams == 0 then
+		return nil
+	end
+
+	local useCallToAction = numberOfStreams <= self.props.callToActionLimit
```

Comment:
if we ever use this widget via widget/factory and `|callToActionLimit=` is set we will get an error here due to comparing number with string

i.e. imo cast to number

possibly need to even do the default handling here if we want to avoid it erroring on bs input (imo not needed, just mentioning the possibility)